src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -34,14 +35,18 @@ public OrderController(ShoppingCartService shoppingCartService,
     }
 
     @PostMapping("/complete")
-    public OrderResponseDto completeOrder(@RequestParam Long userId) {
-        ShoppingCart cart = shoppingCartService.getByUser(userService.get(userId));
+    public OrderResponseDto completeOrder(Authentication authentication) {
+        String userName = authentication.getName();
+        User user = userService.findByEmail(userName).get();
```

Comment:
Maybe you should throw an exception, if you don't find the user by email. Check others